examples/binary_lscp_arcpy.py

```diff
@@ -17,18 +18,18 @@
     sh = logging.StreamHandler(sys.stdout)
     sh.setFormatter(formatter)
     logger.addHandler(sh)
-
+    scriptDir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
can you change this to `script_dir` to make it more "pythonic"?

I assume this was to run it from any directory?

Reply:
Hi. I dont want to commit the changes in the examples/binary_lscp_arcpy.py and binary_mclp_arcpy.py files so I put them in the .gitignore files. However, the .gitignore file does not work here and these changes were submitted. 
Any idea?

Reply:
Can you make a copy of those files and re-name it something else? Then you can have those file locally for yourself.

Then just copy the original files on top of the ones you modified. Then commit those 2 files. That should make those files unchanged.

Or you can try and revert your last couple commits for those specific files if you're familiar with git.